stranded:1.0/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
+
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
+RUN apt-get update
+RUN apt-get -y install gcc git wget
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh 
+RUN conda --version
+ 
+# Configure Services and Port
+COPY start.sh /start.sh
+CMD ["./start.sh"]
+
+RUN conda install -c bioconda samtools
+RUN conda install -c bioconda kallisto=0.44.0
+RUN pip install RSeQC
+RUN pip install git+https://github.com/zoomzoom1011/how_are_we_stranded_here.git
```

Conda environment is again more trouble than it's worth here. Building from an official samtools docker and adding the couple of things we need is probably the best approach.

Additionally, I'm kind of dubious about using your fork of that `how_are_we_stranded_here` repository. Seems like the issue comes down to a Python version problem that we could get around if we reverted to python3.7. I mostly just left your repo in there because the samtools docker is already 20.04 and getting python3.7 set up on that version is more of a chore than its worth.

```suggestion
FROM staphb/samtools:1.16.1

RUN apt update && apt install -y curl python-is-python3 python3-pip git

RUN python -m pip install --upgrade pip && python -m pip install git+https://github.com/zoomzoom1011/how_are_we_stranded_here.git

WORKDIR /opt
ENV KALLISTO_VERSION 0.44.0
RUN curl -sL https://github.com/pachterlab/kallisto/releases/download/v${KALLISTO_VERSION}/kallisto_linux-v${KALLISTO_VERSION}.tar.gz | tar -xz && cp kallisto_linux-v${KALLISTO_VERSION}/kallisto /usr/local/bin

WORKDIR /
COPY Dockerfile .
```